src/main/java/htsjdk/samtools/util/AbstractProgressLogger.java

```diff
@@ -44,35 +46,51 @@ protected AbstractProgressLogger(final String noun, final String verb, final int
      */
     abstract protected void log(String ... message);
 
+    private synchronized void log(final String chrom, final int pos) {
```

Comment:
Why have this take parameters and not just use `this.lastChrom` and `this.lastPos`?